### ISSUE TITLE
Improve help for Publish-Module to include Help for the Path ParameterSets

### ISSUE THREAD

**Issue by kilasuit** [PowerShell/PowerShellGet#406]
The current help for Publish-module doesn't include any examples of using the Path ParameterSets.

**Comment by SydneyhSmith** [comment]
@iricigor yep, that is correct and it should be added everywhere applicable (we try to maintain the docs for all versions), and good catch on the -AllowPrerelease!
